waku/v2/node/wakunode2_test.go

```diff
@@ -318,3 +335,94 @@ func TestDecoupledStoreFromRelay(t *testing.T) {
 	require.Len(t, result.Messages, 1)
 	require.Equal(t, msg.Timestamp, result.Messages[0].Timestamp)
 }
+
+func TestStaticShardingMultipleTopics(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	// Node1 with Relay
+	hostAddr1, err := net.ResolveTCPAddr("tcp", "0.0.0.0:0")
+	require.NoError(t, err)
+	wakuNode1, err := New(
+		WithHostAddress(hostAddr1),
+		WithWakuRelay(),
```

Comment:
If we are writing static sharding tests, we should specify clusterID when we create the Waku node.

Reply:
I see, so the clusterID could actually live, be useful, without sharded topic ?

Reply:
Did not get what you mean

Reply:
I curious about value of setting clusterID (node.WithClusterID ) without having intent to use any shard.

Reply:
But you are using a shard when you are subscribing the pubsubTopic that is part of the clusterid. 
the pubsubTopic format is `/waku/2/rs/<cluster-ID>/<shard>` which means a shard will be included.

Reply:
I see, it is used to create metadata when node instance is created: 

metadata := metadata.NewWakuMetadata(w.opts.clusterID, w.localNode, w.log)
w.metadata = metadata

Metadata won't even start without clusterID != 0:
```
// Start inits the metadata protocol
func (wakuM *WakuMetadata) Start(ctx context.Context) error {
	if wakuM.clusterID == 0 {
		wakuM.log.Warn("no clusterID is specified. Protocol will not be initialized")
		return nil
	}
	.
	.
	.
}
```
I've added WithClusterID and it has no effect on my test. Next interesting thing: node is a member of only one cluster from metadata perspective. Correct ? https://github.com/waku-org/go-waku/pull/1060/commits/4266513a252b3b30480bf2ccad6a85e57d7ee7f0

Reply:
`node is a member of only one cluster from metadata perspective. Correct ?`

Yes, a node can be part of a single cluster. And without clusterID metadata protocol won't be initialized.

As i mentioned above, since these are tests for sharding and as Waku supports 2 types static and auto sharding, we should try to use those formats.